app/views/artists/show.html.erb

```diff
@@ -13,6 +14,10 @@
     <%end%>
 </div>
 
+<% if !current_user.favorited?(current_user.id, @artist.id) %>
+  <h3 class="fav-link"> <%= link_to "FAV", "/favorites/#{@artist.id}/new" %>  </h3>  </br>
+<% end %>
+
```

Comment:
reused code to have a favorite link in the show page. The link only appears if the artist is not already in the user's favorites.